client/components/Header.jsx

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { Navbar, Nav, NavItem, MenuItem, NavDropdown } from 'react-bootstrap';
+
+const Header = () => (
+  <div className="container">
+    <link rel="stylesheet" href="./css/Header.css" />
```

Comment:
when you are loading css in react, it should be ```import './css/Header.css'``` . and you dont define it inside of class
https://stackoverflow.com/questions/39853646/how-to-import-a-css-file-in-a-react-component